src/components/header-system/README.md

```diff
@@ -1,14 +1,384 @@
+                      
+                      <button class="rvt-global-toggle" data-rvt-disclosure-toggle="search" aria-expanded="false">
+                          <span class="rvt-sr-only">Search</span>
+                          <svg xmlns="http://www.w3.org/2000/svg" class="rvt-global-toggle__search" height="16" viewBox="0 0 16 16" width="16">
+                              <path d="M15.71,14.29,10.89,9.47a6,6,0,1,0-1.42,1.42l4.82,4.82a1,1,0,0,0,1.42,0A1,1,0,0,0,15.71,14.29ZM6,10a4,4,0,1,1,4-4A4,4,0,0,1,6,10Z" fill="currentColor"></path>
+                          </svg>
+                      </button>
+
+                      <!-- Search field -->
+
+                      <form action="/search" class="rvt-header-global__search" data-rvt-disclosure-target="search" role="search" method="get" hidden>
+                          <label class="rvt-sr-only" for="search">Search</label>
+                          <div class="rvt-input-group">
+                              <input class="rvt-input-group__input rvt-text-input" type="text" name="q">
+                              <div class="rvt-input-group__append">
+                                  <button class="rvt-button">Search</button>
+                              </div>
+                          </div>
+                      </form>
+                  </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</header>
+```
+
+### Implementing search
+
+By default, the user is sent to the URL `/search` when they submit a search query. The value of their search query is appended to the page URL as the query string parameter `q`.
+
+```
+GET /search?q={search query}
+```
+
+It is up to you to implement a method for handling search queries on your website or application &mdash; most sites use [Google Custom Search](https://developers.iu.edu/resources/google-custom-search/).
+
+If you'd like search to function differently, you can:
+
+- Change the value of the `action` attribute on the search `form` element to redirect the user to a different page on submit 
+- Change the value of the `method` attribute on the search `form` element to `post` to submit the search query via `HTTP POST` instead
+- Change the value of the `name` attribute on the search `input` element to use a different name for the variable containing the search query
+
+To handle submitted search queries yourself without redirecting the user to a new page, use the following JavaScript:
+
+```js
+const searchForm = document.querySelector('[data-rvt-disclosure="search"] form');
+const searchField = searchForm.querySelector('input');
+
+searchForm.addEventListener('submit', event => {
+  event.preventDefault();
+
+  const query = searchField.value;
+
+  // Handle query...
+});
+```
+
+## Secondary navigation
+
+You can add a second level of navigation links beneath the header. This option is useful for section-specific navigation or subsites within a larger website.
```

Comment:
I don't have a problem with this language, but I kind of remember purposefully removing all "subsite" language when we were writing the RFC for the header system last year. Let me see if I can find out why we made those changes. We don't need to let that hold up this PR though.

Reply:
@levimcg Oh interesting, I had no idea. I'll leave it in for now and we can revisit later.